src/app/modules/products/pages/single-product/single-product.page.html

```diff
@@ -0,0 +1,10 @@
+<div *ngIf="product | async as c">
+    <h1>{{ c.name }}</h1>
```

Comment:
We should use desriptive variable names.

Here we can either change c to currentProduct or change product$ to currentProduct$ and then c to product